src/test/kotlin/org/jitsi/jibri/helpers/Helpers.kt

```diff
@@ -43,6 +43,10 @@ fun <T> within(duration: Duration, func: () -> T): T {
 }
 
 fun Int.seconds(): Duration = Duration.ofSeconds(this.toLong())
+val Int.seconds: Duration
```

Comment:
Do we need both the `fun` and the `val`?

Reply:
Nope, fixed.